### ISSUE TITLE
plugin: cactbot can't detect language when OverlayPlugin loaded before FFXIV_ACT_Plugin

### ISSUE THREAD

**Issue by quisquous** [quisquous/cactbot#594]
I think this is the only ordering constraint about loading that cactbot creates.  The language isn't really needed until later when the overlay finally is ready to start, and so this should be deferred so that it would work regardless of the order of plugins.

**Comment by quisquous** [comment]
I don't think there's anything that can be done here.

**Comment by ngld** [comment]
Can you check if this is still an issue?

A while ago, I fixed several issues in OverlayPlugin caused by loading it before the FFXIV plugin. As a side effect, event sources are started only after *all* plugins  have been loaded. As long as you don't access the FFXIV plugin before the `Start` method is called, you should be fine.

**Comment by quisquous** [comment]
You're quite right, this does work just fine now.  Still have to load CactbotOverlay.dll after OverlayPlugin.dll, but the FFXIV plugin vs OverlayPlugin.dll is now ok to switch.

**Comment by quisquous** [comment]
Actually, scratch this.  If I do this, this is detected ok in the log output for cactbot, but no cactbot overlays show up.
